protocols/identify/src/behaviour.rs

```diff
@@ -89,7 +89,13 @@ pub struct Config {
     /// The initial delay before the first identification request
     /// is sent to a remote on a newly established connection.
     ///
-    /// Defaults to 500ms.
+    /// Defaults to 0ms.
+    #[deprecated(
+        since = "0.42.1",
```

Comment:
I think removing this version more sense. Then we can't get it wrong and it will simply trigger once released.